pkg/provider/azure_loadbalancer.go

```diff
@@ -1543,11 +1570,20 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 
 	lbName := *lb.Name
 	lbResourceGroup := az.getLoadBalancerResourceGroup()
-	isIPv6 := utilnet.IsIPv6String(service.Spec.ClusterIP)
-	lbBackendPoolID := az.getBackendPoolID(lbName, az.getLoadBalancerResourceGroup(), getBackendPoolName(clusterName, isIPv6))
+	lbBackendPoolIDs := map[bool]string{
```

Comment:
can we extract a function instead? e.g. generateBackendpoolId

Reply:
Updated